subworkflows/local/refine_fdr.nf

```diff
@@ -36,16 +36,17 @@ workflow REFINE_FDR {
         ch_versions = ch_versions.mix(MHCFLURRY_PREDICTPSMS.out.versions)
 
         // Filter psm results by shrinked search space
+        // TODO: Check if filtering works properly when reevaluating this subworkflow
```

Comment:
There is a TODO here, is there an issue generated for this or does it need to be included for this PR?

Reply:
There is not an issue on this since it is more a backlog bullet point right now 😁